prepare-gcs.js

```diff
@@ -0,0 +1,69 @@
+// require
+var fs = require('fs');
+var pckg = require('./package.json');
+
+// helper
+var copy = function(srcDir, dstDir) {
```

Comment:
Why not just use the [copy](https://www.npmjs.com/package/copy) package?

Reply:
I copied it only from another repo from us: https://github.com/tourstream/fti-portal-styleguide/blob/master/prepare-gcs.js

@dasrick What do you think of the idea, to create an own npm library for that?